### PR TITLE
(EAI-497) DevCenter reference links should be labeled Article

### PR DESCRIPTION
**Pull request by nlarew**
Jira: (EAI-497) DevCenter reference links should be labeled Article

## Changes

- Switch from Blog to Article
- Use variants directly from LG chat instead of hardcoding

### CODE REVIEW

File: packages/mongodb-chatbot-ui/src/messageLinks.tsx [hunk 1]
```diff
@@ -42,3 +26,12 @@ export function formatReferences(
     return richLinkProps;
   });
 }
+
+export function getMessageLinks(
```
Comment (mongodben) [inline packages/mongodb-chatbot-ui/src/messageLinks.tsx:30]: should there be tests for this?

File: packages/chatbot-server-mongodb-public/src/processors/makeMongoDbReferences.ts [hunk 2]
```diff
@@ -75,6 +75,7 @@ export function mongodbReferenceType(
       case "practical-aggregations-book":
         return "Book";
       case "devcenter":
+        return "Article";
```
Comment (mongodben) [inline packages/chatbot-server-mongodb-public/src/processors/makeMongoDbReferences.ts:78]: relevant tests now failing b/c of this change. pls update test to match

**Review by mongodben — APPROVED** [review]
lgtm % changes noted in review

File: packages/mongodb-chatbot-ui/src/messageLinks.tsx (hunk 1 above)
Comment (nlarew) [reply packages/mongodb-chatbot-ui/src/messageLinks.tsx:30]: Yeah - I actually have some tests for this file in another PR. I'll port them over to this one

File: packages/chatbot-server-mongodb-public/src/processors/makeMongoDbReferences.ts (hunk 2 above)
Comment (nlarew) [reply packages/chatbot-server-mongodb-public/src/processors/makeMongoDbReferences.ts:78]: Updated

**Pull request merged**